start-script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Maybe add a comment about the usage of this file? Such as "This is the entry script for docker container. It does ..." and so on.